src/main/java/com/microsoft/azure/util/AzureCredentialsBinding.java

```diff
@@ -29,20 +29,20 @@
  * and pipeline using Credentials Binding plugin.
  * There're two ways to construct this binding:
  * 1. With defaults, which will read specified service principal into four predefined environment
- *    variables: AZURE_SUBSCRIPTION_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_TENANT_ID.
- *    Sample pipeline code:
- *      withCredentials([azureServicePrincipal('credentials_id')]) {
- *          sh 'az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET -t $AZURE_TENANT_ID'
- *      }
+ * variables: AZURE_SUBSCRIPTION_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_TENANT_ID.
```

Comment:
Oh, that's folded by the IntelliJ auto-format-code. I updated with proper &lt;pre&gt; &amp; &lt;code&gt; around the code samples to prevent space removal.